### DISCUSSION TITLE
Fix Dropdown documentation

### DISCUSSION THREAD

**Discussion by promethey** [twbs/bootstrap#36374] [General]
Hi. Please fix Dropdown documentation:
1. [Single button first example](https://getbootstrap.com/docs/5.1/components/dropdowns/#single-button) don`t use ```<button>``` element;
2. Add examples for dark variant (and maybe light) of dropdown toggle button;
3. Change direction [dropup](https://getbootstrap.com/docs/5.1/components/dropdowns/#single-button) classname like a flex utility: ```.dropup``` -> ```.droptop```;

P.S. Issues persisted in v5.2 as well

Thanks!

**Comment by mdo** [comment]
Sorry to say, but I honestly don't have a clue what you're asking us to do 😅

**Reply by promethey** [reply]
I attach a more detailed explanation)

1. I think the example is missing the use of .dropdown-item as a button element.

2. Dark and light variations of .dropdown-toggle are missing in the examples (Why?).  The .btn-dark and .btn-light classes are ready.

3. Directions classes are not named systematically: .dropstart, .dropend, .dropup?  I think it would be great to rename the class name from .dropup to .droptop (for a better understanding: start, end, top)

Thanks!
